src/main/java/Suv.java

```diff
@@ -0,0 +1,94 @@
+import java.util.Scanner;
+
+
+public class Suv {
+    final static int DEADLINE_KEYWORD_END_INDEX = 8;
+    final static int TODO_KEYWORD_END_INDEX = 4;
+    final static int EVENT_KEYWORD_END_INDEX = 5;
+    final static int TO_KEYWORD_END_INDEX = 2;
+    final static int FROM_KEYWORD_END_INDEX = 4;
+
+    public static void main(String[] args) {
+        String name = "Suv";
+        Scanner in = new Scanner(System.in);
+        Task[] tasks = new Task[100];
+        int taskIndex = 0;
+
+        System.out.println("____________________________________________________________\n" +
```

Comment:
as this printing dividing line command appears many many times in the code, can consider changing the line into a void function, so that it looks less messy